ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/Queries.scala

```diff
@@ -67,6 +68,7 @@ sealed abstract class Queries(tablePrefix: String) {
         (contract_id $contractIdType NOT NULL CONSTRAINT ${tablePrefixFr}contract_k PRIMARY KEY
         ,tpid $bigIntType NOT NULL REFERENCES $templateIdTableName (tpid)
         ,${jsonColumn(sql"key")}
+        ,key_hash $keyHashColumn
```

Comment:
I  expect it’s still faster even on postgres since indexing a short text should be faster than indexing a json value and if we can, I think we’re better off keeping the two schemas close to each other. Fully agree on removing the other index.